thermo/gpumd/io.py

```diff
@@ -249,6 +249,61 @@ def import_trajectory(filename='movie.xyz', in_file=None, atom_types=None):
 # Write Related
 #########################################
 
+def create_kpoints(special_points, npoints, atom_c):
+    """
+     Creates the file "kpoints.in", which specifies the kpoints needed for src/phonon
+
+    Args:
+        atom_c (ase.Atom):
+            atom to build
+
+        special_points (str):
+            specify special points for kpoint generation
+
+        npoints (int):
+            specifies the number of kpoints to generate
+    """
+
+    path = atom_c.cell.bandpath(special_points, npoints)
```

Comment:
```suggestion
    path = atoms.cell.bandpath(special_points, npoints)
```
Updated variable name to match above